scrapers/tv.py

```
@@ -1,18 +1,22 @@
 
 import dramatiq
+import httpx
 from beanie import BulkWriter
+from beanie.odm.operators.update.general import Set
 from ipytv import playlist
 from ipytv.channel import IPTVAttr
 
 from db import schemas, crud
-from db.models import TVStreams
+from db.models import TVStreams, MediaFusionTVMetaData
+from db.schemas import TVMetaProjection
```

Ruff

`db.schemas.TVMetaProjection` imported but unused

Remove unused import: `db.schemas.TVMetaProjection`

(F401)